config/packages/security.yaml

```diff
@@ -36,5 +36,5 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        # - { path: ^/admin, roles: ROLE_ADMIN }
+        - { path: ^/admin, roles: ROLE_ADMIN }
```

Comment:
The path of admin is safe but an basic user can see the button to go to the backoffice in account route.